pkg/neorpc/result/raw_notary_pool.go

```diff
@@ -0,0 +1,46 @@
+package result
+
+import (
+	"encoding/json"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// RawNotaryPool represents a result of `getrawnotarypool` RPC call.
+// The structure consist of `Hashes`. `Hashes` field is a map, where key is
+// the hash of the main transaction and value is a slice of related fallback
+// transaction hashes.
+type RawNotaryPool struct {
```

Comment:
Typo in the commit body: s/`returns hashes from all the verified transactions`/`returns hashes of all the verified transactions`.

Reply:
Fixed.